test/fennec_test.exs

```diff
@@ -9,7 +9,8 @@ defmodule FennecTest do
     import Fennec.Test.Helper.Server, only: [configuration: 1]
     %{address: a, port: p} = configuration("Fennec (local)")
     Fennec.UDP.start_link(ip: a, port: p)
-    {:ok, alice} = Jerboa.Client.start(server: {a, p})
+    Application.put_env(:fennec, :secret, "abc") 
```

Comment:
Could you put examples of all introduced environment values in config files (they may be commented). Or document them in docs for relevant public function. So that we don't lose track of them.